lib/sequelize.js

```diff
@@ -1109,7 +1109,8 @@ class Sequelize {
           // Rollback transaction if not already finished (commit, rollback, etc)
           // and reject with original error (ignore any error in rollback)
           return Promise.try(() => {
-            if (!transaction.finished) return transaction.rollback().catch(() => {});
+            if (!transaction.finished) return transaction.rollback().catch(() => {
+            });
```

Comment:
I think the previous format was better here, please revert

Reply:
My IDE do this automatically. I will revert. Sorry about that.